jobs/health.rb

```diff
@@ -50,6 +50,7 @@
     {name: 'offender-events', versionUrl: 'https://offender-events.prison.service.justice.gov.uk/info', url: 'https://offender-events.prison.service.justice.gov.uk/health'},
     {name: 'case-notes-to-probation', versionUrl: 'https://health-kick.prison.service.justice.gov.uk/https/case-notes-to-probation.prison.service.justice.gov.uk/info', url: 'https://health-kick.prison.service.justice.gov.uk/https/case-notes-to-probation.prison.service.justice.gov.uk'},
     {name: 'probation-teams', versionUrl: 'https://probation-teams.prison.service.justice.gov.uk/info', url: 'https://probation-teams.prison.service.justice.gov.uk/health'},
+    {name: 'prison-to-probation-update', versionUrl: 'https://health-kick.prison.service.justice.gov.uk/https/prison-to-probation-update.prison.service.justice.gov.uk/info', url: 'https://health-kick.prison.service.justice.gov.uk/https/prison-to-probation-update.prison.service.justice.gov.uk'},
```

Comment:
Doesn't need to go via health kick (my bad).  Can be removed provided you add `/health` onto the end